extensions/wikia/Recirculation/templates/client/rail.mustache

```diff
@@ -1,13 +1,14 @@
-<section class="recirculation-rail module">
-	<h2>{{ title }}</h2>
+<section class="recirculation-rail module group-{{ group }}">
+	<h2>{{{ title }}}</h2>
```

Comment:
This looks fishy. Is there any reason to have the unescaped version other than to accept the '<br>' as added by `formatTitle`?